commands/miner_daemon_test.go

```diff
@@ -209,23 +229,33 @@ func TestMinerSetPrice(t *testing.T) {
 func TestMinerCreateSuccess(t *testing.T) {
 	tf.IntegrationTest(t)
 
-	d1 := makeTestDaemonWithMinerAndStart(t)
-	defer d1.ShutdownSuccess()
-	d := th.NewDaemon(t, th.KeyFile(fixtures.KeyFilePaths()[2])).Start()
-	defer d.ShutdownSuccess()
-	d1.ConnectSuccess(d)
-	var wg sync.WaitGroup
-	wg.Add(1)
-	go func() {
-		miner := d.RunSuccess("miner", "create", "--from", fixtures.TestAddresses[2], "--gas-price", "1", "--gas-limit", "100", "200")
-		addr, err := address.NewFromString(strings.Trim(miner.ReadStdout(), "\n"))
-		assert.NoError(t, err)
-		assert.NotEqual(t, addr, address.Undef)
-		wg.Done()
+	ctx, env := fastesting.NewTestEnvironment(context.Background(), t, fast.FilecoinOpts{})
```

Comment:
In the process of debugging a miner creation failure, I converted this to a FAST test.

Reply:
god's work 🙏